docs/application-connector/docs/assets/001-application-connector.svg

```diff
@@ -1,2 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You edited an outdated version of the diagram - we don't use the term "Remote Environment" anymore.

Reply:
Thanks for spotting that 😨